packages/console/src/monaco/MonacoProviders.tsx

```diff
@@ -84,6 +87,40 @@ class MonacoProviders extends PureComponent<
     }
   }
 
+  /**
+   * Converts an LSP document range to a monaco range
+   * Accounts for LSP indexing from 0 and monaco indexing from 1
+   *
+   * @param range The LSP document range to convert
+   * @returns The corresponding monaco range
+   */
+  static lspToMonacoRange(range: DocumentRange): monaco.IRange {
+    const { start, end } = range;
+
+    // Monaco expects the columns/ranges to start at 1. LSP starts at 0
+    return {
+      startLineNumber: start.line + 1,
+      startColumn: start.character + 1,
+      endLineNumber: end.line + 1,
+      endColumn: end.character + 1,
+    };
+  }
+
+  /**
+   * Converts a monaco position to an LSP position
+   * Accounts for LSP indexing from 0 and monaco indexing from 1
+   *
+   * @param position The monaco position
+   * @returns The corresponding LSP position
+   */
+  static monacoToLspPosition(position: monaco.IPosition): Position {
```

Comment:
Would be fairly easy to add tests for these, or add a test that verifies the positions in the provider.

Reply:
Added tests for the range/position transforms